cmd/goose/go_sqlite3.go

```diff
@@ -0,0 +1,10 @@
+// +build !nosqlite3
+
+package main
+
+// including go-sqlite3
```

Comment:
Aye. It's why I left it in, to keep `go lint` nice and quiet. Although, we don't _need_ to always abide by what it says.